pkg/cases/handler.go

```diff
@@ -8,14 +8,106 @@ import (
 type CaseHandler struct {
 }
 
-// @Summary Get case by case id
+// @Summary New a case
 // @Description
+// @Tags Case
 // @Produce json
-// @Param  id path uint true "case ID"
+// @Param caseDetail body string true "patient ID, doctor ID and department name"
 // @Success 200 {object} api.ReturnedData{data=Case}
```

Comment:
And the returned data contains not a case object

Reply:
emmm, so what should i return? a msg?

Reply:
The returned data is a `api.ReturnedData`, it contains a fixed msg entry, but what is in the data entry depends. If you do not want to return anything other than the message, simply omit the `data=Case` part. (or you can try `data=nil`, I'm not sure whether it works or not)